tests/php-code-style/.php-cs-fixer.dist.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+$APPROOT=dirname(__DIR__, 2);
+
+echo $APPROOT;
+$finder = PhpCsFixer\Finder::create()
+    ->exclude('oql')
+	->in($APPROOT.'/addons')
+	->in($APPROOT.'/application')
+	->in($APPROOT.'/core')
+	->in($APPROOT.'/datamodels')
+	->in($APPROOT.'/dictionaries')
+	->in($APPROOT.'/pages')
+	->in($APPROOT.'/portal')
+	->in($APPROOT.'/setup')
+	->in($APPROOT.'/sources')
+	->in($APPROOT.'/synchro')
+	->in($APPROOT.'/tests')
+    ->in($APPROOT . '/webservices')
```

Comment:
This one looks off..

```suggestion
    ->in($APPROOT.'/webservices')
```

Reply:
inception... i should apply code style to code style PHP file... :)

Reply:
Not fixed yet, please unresolved until it is.

Reply:
it should be fine now